orienteer-core/src/main/java/org/orienteer/core/component/visualizer/AbstractSimpleVisualizer.java

```diff
@@ -54,4 +53,8 @@ public boolean isExtended() {
 		return supportedTypes;
 	}
 
+	@Override
+	public <V> Component createComponentForFiltering(String id, IModel<OProperty> propertyModel, IModel<V> valueModel) {
+		return new Label(id, Model.of("No realisation for filtering component in current visualizer: " + getName()));
```

Comment:
By default it should return null - which means that no filtering capabilities available for current property.